Tribes/src/app/_components/notifications/notifications.component.ts

```diff
@@ -25,31 +26,33 @@ export class NotificationsComponent implements OnInit {
           this.generateListToDisplay();
       }
     });
+    this.buildingService.updateRessourceByConstruction.subscribe({
+      next: () => {
+        this.emptyMessage = undefined;
```

Comment:
Whooohoooo... I have not noticed this! Why is this needed?

Reply:
This is a message, which is shown in the sidebar if we don't have any building under construction. But if we pressed the button to add a new one, or to upgrade, we don't want to see anymore this message.